itests/fevm_test.go

```diff
@@ -48,22 +46,60 @@ func TestFEVMBasic(t *testing.T) {
 
 	idAddr, err := address.NewIDAddress(result.ActorID)
 	require.NoError(t, err)
-	t.Logf("actor ID address is %s", idAddr)
 
-	// invoke the contract with owner
-	{
-		entryPoint, err := hex.DecodeString("f8b2cb4f")
-		require.NoError(t, err)
+	return fromAddr, idAddr
+}
 
-		inputData, err := hex.DecodeString("000000000000000000000000ff00000000000000000000000000000000000064")
-		require.NoError(t, err)
+func invokeContract(t *testing.T, ctx context.Context, client *kit.TestFullNode, fromAddr address.Address, idAddr address.Address, funcSignature string, inputData []byte) []byte {
```

Comment:
As @arajasek mentioned, it would be nice to make these helpers available to other tests